### PR TITLE
Allow Symfony 4

### PR DESCRIPTION
**Pull request by mmaarriiuuss**
I am targeting this branch, because it's just a simple composer and travis update.

Updated the composer.json and travis.yml for Symfony Flex.
#SymfonyConHackday2017

## Changelog

```markdown
### Fixed
- It is now allowed to install Symfony 4
```

### CODE REVIEW

**Comment by greg0ire** [comment]
Please associate your pitechplus email address with your Github account, or change the email in your commits to an address already associated with it.

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +58,10 @@ matrix:
       env: SONATA_BLOCK='dev-master as 3'
     - php: '7.1'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: '7.1'
+      env: SYMFONY_VERSION=^3.0
+    - php: '7.1'
+      env: DEPENDENCIES=beta
```
Comment (greg0ire) [inline .travis.yml]: We manage this file with https://github.com/Soullivaneuh/sonata-dev-kit

Comment (jordisala1991) [reply .travis.yml]: https://github.com/sonata-project/dev-kit

**Comment by greg0ire** [comment]
We are already testing sf4, thanks to an inline alias: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/.travis/before_install_test.sh#L12

File: composer.json [hunk 2]
```diff
@@ -49,8 +49,8 @@
         "sensio/generator-bundle": "^3.1",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
         "sonata-project/intl-bundle": "^2.2.4",
-        "symfony/phpunit-bridge": "^3.3",
-        "symfony/yaml": "^2.8 || ^3.2"
+        "symfony/phpunit-bridge": "^3.3 || ^4.0",
+        "symfony/yaml": "^2.8 || ^3.2 || ^4.0"
```
Comment (greg0ire) [inline composer.json]: This part is ok, we could keep it.

**Comment by greg0ire** [comment]
https://twitter.com/greg0ire/status/931826774270410753

**Comment by greg0ire** [comment]
Labelling as pedantic since sf4 is not out, doing a release will have no effect I think.

File: composer.json [hunk 3]
```diff
@@ -24,21 +24,21 @@
...
+        "symfony/class-loader": "^2.8 || ^3.2 || ^4.0",
+        "symfony/config": "^2.8 || ^3.2 || ^4.0",
+        "symfony/console": "^2.8 || ^3.2 || ^4.0",
+        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
+        "symfony/expression-language": "^2.8 || ^3.2 || ^4.0",
+        "symfony/form": "^2.8 || ^3.2 || ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
+        "symfony/property-access": "^2.8 || ^3.2 || ^4.0",
+        "symfony/routing": "^2.8 || ^3.2 || ^4.0",
+        "symfony/security-acl": "^2.8 || ^3.0 || ^4.0",
```
Comment (jordisala1991) [inline composer.json]: this bundle is not following SF versioning anymore.

**Comment by OskarStark** [comment]
Close/Reopen for triggering StyleCi

**Comment by Soullivaneuh** [comment]
@greg0ire It's a patch. This package can't be installed with SF4 without this PR. Am I wrong?

**Comment by greg0ire** [comment]
Right now that would probably not warrant a release because sf4 is not released so pedantic? But in the future it should indeed be patch, so let's go with patch probably, doesn't hurt.

**Comment by Soullivaneuh** [comment]
> But in the future it should indeed be patch, so let's go with patch probably, doesn't hurt

The problem is it will be forgotten in the future. :-)

**Comment by greg0ire** [comment]
Yeah, good call!

**Comment by greg0ire** [comment]
I added one

**Pull request merged**